app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split()
+    cp = command_list[0]
```

Comment:
```
    cp = command_list[0]
    source_file = command_list[1]
    dest_file = command_list[2] 
```
it must be after len(command_list) == 3, because code will be broken if len(command_list) != 3